ci-operator/config/open-cluster-management/multicloud-operators-foundation/open-cluster-management-multicloud-operators-foundation-main.yaml

```diff
@@ -64,6 +68,15 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: sonarcloud
+  commands: |
+    export SELF="make -f /opt/build-harness/Makefile.prow"
+    make -f /opt/build-harness/Makefile.prow sonar/go/prow
+  container:
+    from: open-cluster-management_builder_go1.15-linux-amd64
```

Comment:
This needs to be "from: src". The "open-cluster-management_builder_go1.15-linux-amd64" doesn't have your source code injected into it.

The "src" image is automatically built by Prow by taking the "build_root" image and injecting your source code into it.